src/gui/widgets/Fader.cpp

```diff
@@ -330,14 +317,14 @@ void Fader::setPeak_R( float fPeak )
 // update tooltip showing value and adjust position while changing fader value
 void Fader::updateTextFloat()
 {
-	if( ConfigManager::inst()->value( "app", "displaydbfs" ).toInt() && m_displayConversion )
+	if( ConfigManager::inst()->value( "app", "displaydbfs" ).toInt() && ( m_conversionFactor == 100.0 ) )
```

Comment:
The parentheses around the comparison is not needed.

Reply:
`m_conversionFactor == 100.0` doesn't always imply it's a volume fader, but this look out of scope of this PR.